x-pack/plugins/canvas/public/apps/workpad/workpad_app/workpad_telemetry.tsx

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState, useEffect } from 'react';
+import { connect } from 'react-redux';
+// @ts-ignore
```

Comment:
nit: please add a comment as to why this was ignored, (e.g. `local untyped file` or `library without @typed definition`